pkg/sriov/poolconfig.go

```diff
@@ -255,6 +255,33 @@ func (builder *PoolConfigBuilder) WithMaxUnavailable(maxUnavailable intstrutil.I
 	return builder
 }
 
+// WithRdmaMode method sets rdmaMode to shared/exclusive.
+func (builder *PoolConfigBuilder) WithRdmaMode(mode string) *PoolConfigBuilder {
```

Comment:
nit: RDMA is acronym so worth using capital letters 
```suggestion
func (builder *PoolConfigBuilder) WithRDMAMode(mode string) *PoolConfigBuilder {
```

Reply:
done. PTAL